src/index.js

```diff
@@ -22,7 +22,9 @@ if (window.location.href.split(':').includes('file')) {
       // Blackbox
       `/^Cannot read properties of null (reading 'CodeMirror')$/ `,
       //Firefox interdit aux addOns de conserver des références fortes aux objets DOM après la destruction de leur document parent.
-      `/^can't access dead object$/ `
+      `/^can't access dead object$/ `,
```

Comment:
je viens de tilter que l'erreur sentry a été généré il y a 6 jours ici : https://sentry.incubateur.net/organizations/betagouv/issues/40307/?project=68&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=90d

Reply:
Est-ce que c'est à cause de l'espace à la fin de la chaîne ?
Sinon j'ai vu une autre façon d'ignorer les envoies à sentry
https://docs.sentry.io/platforms/javascript/guides/react/configuration/filtering/

Reply:
ah oui j'avais pas gaffe à l'espace, je ne pense pas que ce soit lié à l'apostrophe

Reply:
à tenter sans les espaces effectivement à la fin